.travis.yml

```diff
@@ -42,6 +42,7 @@ env:
     - REDIS_CLUSTER_IP_TEST=localhost
     - SESSION_RESET_THRESHOLD_TEST=2000
     - SUBSEQUENT_ALERT_MESSAGE_THRESHOLD_TEST=1000
+    - RADAR_WINDOW_SIZE_SECONDS=2
```

Comment:
This might be your bug. You need `_TEST` at the end of this. Otherwise, when running the tests on Travis, you'll end up with `undefined` when it calls `helpers.getEnvVar('RADAR_WINDOW_SIZE_SECONDS')`.